### ISSUE TITLE
VSCode Extension cannot find packages indicated in tsconfig.json

### ISSUE THREAD

**Issue by Icexist** [mdx-js/mdx-analyzer#442]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

v1.8.6

### Steps to reproduce

When using the MDX plugin in VSCode, set the tsconfig.json file to have any number of plugins and try to open an mdx file. 

It works just fine with the 1.8.5 version (installed via "install other version" in vscode)

my `tsconfig.json` file:

```json
{
  "include": ["**/*"],
  "compilerOptions": {
    // …
  },
  "mdx": {
    "plugins": [
      "remark-frontmatter",
      "remark-gfm",
      "remark-math",
      "rehype-katex"
    ]
  }
}
```

### Expected behavior

MDX should work as expected.

### Actual behavior

The MDX plugin crashes:

> Error: Cannot find package 'remark-frontmatter' imported from /home/xxxx/project
>     at new n (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:858:18648)
>     at Sq (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:7076)
>     at Xle (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:7581)
>     at Eq (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:8808)
>     at Gv (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:8998)
>     at Lq (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:10157)
>     at async Kv (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:860:9400)
>     at async Promise.all (index 0)
>     at async s (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:865:18826)
>     at async sae (/home/xxxx/.vscode-server/extensions/unifiedjs.vscode-mdx-1.8.6/out/language-server.js:96:5817) {
>   code: 'ERR_MODULE_NOT_FOUND'
> }
> 
> Node.js v18.18.2
> [Error - 4:13:18 PM] Server process exited with code 1.
> [Error - 4:13:18 PM] The MDX server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.

### OS

Windows 10, running VSCode remote in a WSL2 instance. MDX plugin runs on the WSL instance.

**Comment by remcohaszing** [comment]
In version 1.8.0-1.8.5 there was a bug that caused the language server to ignore the `plugins` field. So it defaulted to `["remark-frontmatter", "remark-gfm"]`. That’s why this worked for you kind of by accident.

You should specify `"remark-math"`, because it adds support for **parsing** the math syntax, but not `"rehype-katex"`, as this only **transforms** the AST.

Do you have all those plugins installed in `/home/xxxx/project`? In other words: Does `/home/xxxx/project/node_modules/remark-frontmatter` exist and can you import `remark-frontmatter` from `/home/xxxx/project/some-script.mjs`? with the following contents:

```js
import 'remark-frontmatter'
```

**Comment by Icexist** [comment]
Verify that those (and more) are in node_modules: 
![image](https://github.com/mdx-js/mdx-analyzer/assets/5529540/59ce9df1-1e7d-43bb-a33a-4e47c57240b7)

I'm running Docusaurus, so i'm not fully clear on how to test the .mjs file, but these extensions work just fine in Docusaurus. 

Still have the same problems after removing `rehype-katex` - the extension can't find the modules.

Here's a fresh repository that shows the issue for me: https://github.com/Icexist/mdx-ext-debug

**Comment by Icexist** [comment]
As a note, that repo was created on MacOS and showed the issue in VSCode there, so this isn't limited to Windows.

**Comment by remcohaszing** [comment]
Thanks for the reproduction. I’m able to reproduce this on Linux too.

**Comment by remcohaszing** [comment]
This was fixed in version [1.8.7](https://github.com/mdx-js/mdx-analyzer/releases/tag/vscode-mdx@1.8.7) :tada:
